PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,27 +1,18 @@
 ## Description
-<!-- Please explain the changes you made here. -->
+
 
 ## Testing done
-<!-- Please explain local testing done, this can include
-automated testing, manual testing, cross browser testing -->
 
-## Testing Plan
-<!-- Please explain testing planned this can include
-automated testing, manual testing, cross browser testing -->
 
 ## Screenshots
-<!-- Please provide screenshots of UI changes if applicable -->
-
-## Acceptance Criteria (Definition of Done)
 
-#### Unique to this PR
-<!-- This would be a good place to include feature flag check item and info, specific dashboards and instrumentation check item and info -->
-- [ ] _Replace this line with individual tasks unique to your PR_
 
-#### Applies to all PRs
+## Acceptance criteria
+- [ ]
 
-- [ ] Appropriate logging
+## Definition of done
+- [ ] Events are logged appropriately
 - [ ] Documentation has been updated, if applicable
-- [ ] Provide link to originating GitHub issue, or connected to it via ZenHub
-- [ ] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
-- [ ] Provide which alerts would indicate a problem with this functionality (if applicable)
+- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
+- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
+- [ ] Alerts are provided which would indicate a problem with this functionality (if applicable)
```

Comment:
I'm still not really sure what this line really means. Can someone explain it to me?

Reply:
Good question. I was originally thinking logging errors to Sentry, but it does seem like that's more or less covered in `Events are logged appropriately`.

Reply:
This template is based on the template for vets-api, and it seems like this may not be relevant for vets-website and can be removed.